### DISCUSSION TITLE
Please Help, SWAR STOPPED WORKING

### DISCUSSION THREAD

**Discussion by MarcelodeFA** [swar/Swar-Chia-Plot-Manager#1069] [Q&A]
Hi guys, my swar just stopped working since I downloaded version 1.1.6, already changed the chia exe locator, but the swar just stays empty of plots, and "running"
I noticed that it keeps opening the Chia GUI over and over
Already deleted chia mainnet and swar's files, and reinstalled both, made new configurations for swar. But didnt work, same error.
Actually it doesn't give me any errors message, it just keeps empty of plots.
![Captura de Tela (33)](https://user-images.githubusercontent.com/85201643/120400776-f4dfc500-c314-11eb-99aa-02e09d5f5241.png)

**Comment by MarcelodeFA** [comment]
[config - Copia.txt](https://github.com/swar/Swar-Chia-Plot-Manager/files/6581007/config.-.Copia.txt)
My swar's configurations.

**Comment by mikessb** [comment]
the executable command   in the config file is "chia.exe" , not  "Chia.exe"     -_-!

**Comment by FlyingToto** [comment]
indeed you picked the wrong executable, it is confusing because they are called the same but the one you selected is the one that open the gui not the one to plot check the windows example in your config file... you want to pick the one under the subfolders resources\app.asar.unpacked\daemon\chia.exe
